### PR TITLE
feat: use badges for registration status in PA table AB#27937

### PR DESCRIPTION
**Pull request by arsforza**
## Describe your changes

The status column in the PA table will show a color coded badge instead of a simple text.
Also, the colors used by the system have been updated to align with the ones in design system.

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] I have added tests wherever relevant
- [x] I have made sure that all automated checks pass before requesting a review
- [x] I do not need any deviation from our PR guidelines

### CODE REVIEW

File: interfaces/Portal/src/theme/variables.scss [hunk 1]
```diff
@@ -1,95 +1,105 @@
...
+  --ion-color-light-tint: var(--ion-color-fiveten-neutral-100-tint);
 
   /** purple button **/
-  --ion-color-purple-button: #4f22d7;
+  --ion-color-purple-button: var(--ion-color-fiveten-purple-700);
 
   /** accordion text **/
-  --ion-color-accordion-text: #0f1218;
+  --ion-color-accordion-text: var(--ion-color-fiveten-neutral-900);
 }
```
Comment (elwinschmitz) [inline interfaces/Portal/src/theme/variables.scss:105]: Are these actually used somewhere still?

**Review by elwinschmitz — APPROVED** [review]

**Comment by ashishkumarbasak-accenture** [comment]
@arsforza should we merge this MR to feature branch `feat.combine-inclusion-registrationvalidation`?

**Comment by arsforza** [comment]
> @arsforza should we merge this MR to feature branch `feat.combine-inclusion-registrationvalidation`?

@ashishkumarbasak-accenture I would love to, but the tests are failing and I can't get them to work on my local env to fix them

**Comment by ashishkumarbasak-accenture** [comment]
> work on my local env to

I think @Piotrk39 already fixed the e2e tests, so you just have to update (rebase) your branch with the feature branch and then I think tests will be green. Thanks

**Review by ashishkumarbasak-accenture — APPROVED** [review]

**Pull request merged**